
From: Roland Dreier <roland@topspin.com>

Add qp_type to struct ib_qp.

Signed-off by: Sean Hefty <sean.hefty@intel.com>
Signed-off-by: Roland Dreier <roland@topspin.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/infiniband/core/verbs.c       |    1 +
 25-akpm/drivers/infiniband/include/ib_verbs.h |    1 +
 2 files changed, 2 insertions(+)

diff -puN drivers/infiniband/core/verbs.c~infiniband-core-add-qp_type-to-struct-ib_qp drivers/infiniband/core/verbs.c
--- 25/drivers/infiniband/core/verbs.c~infiniband-core-add-qp_type-to-struct-ib_qp	Wed Jan 12 16:31:58 2005
+++ 25-akpm/drivers/infiniband/core/verbs.c	Wed Jan 12 16:31:58 2005
@@ -132,6 +132,7 @@ struct ib_qp *ib_create_qp(struct ib_pd 
 		qp->srq	       	  = qp_init_attr->srq;
 		qp->event_handler = qp_init_attr->event_handler;
 		qp->qp_context    = qp_init_attr->qp_context;
+		qp->qp_type	  = qp_init_attr->qp_type;
 		atomic_inc(&pd->usecnt);
 		atomic_inc(&qp_init_attr->send_cq->usecnt);
 		atomic_inc(&qp_init_attr->recv_cq->usecnt);
diff -puN drivers/infiniband/include/ib_verbs.h~infiniband-core-add-qp_type-to-struct-ib_qp drivers/infiniband/include/ib_verbs.h
--- 25/drivers/infiniband/include/ib_verbs.h~infiniband-core-add-qp_type-to-struct-ib_qp	Wed Jan 12 16:31:58 2005
+++ 25-akpm/drivers/infiniband/include/ib_verbs.h	Wed Jan 12 16:31:58 2005
@@ -659,6 +659,7 @@ struct ib_qp {
 	void                  (*event_handler)(struct ib_event *, void *);
 	void		       *qp_context;
 	u32			qp_num;
+	enum ib_qp_type		qp_type;
 };
 
 struct ib_mr {
_
